config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
Overriding any setting with docker should be achieved using environment variables / a docker compose override file. This is the way it will be done in production.

See /var/www/vhosts/c2corgv6-images/private/v6_images/docker-compose.override.yml on demo server.

Are we gaining something in using property files in addition to compose files?